presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/JdbcUtil.java

```diff
@@ -27,4 +38,27 @@ public static void enableStreamingResults(Statement statement)
             statement.unwrap(com.mysql.jdbc.Statement.class).enableStreamingResults();
         }
     }
+
+    public static JDBCType jdbcType(Type type)
```

Comment:
Since method is only used in one place, keep it in the same class where it's used